sql/signal_handler.cc

```diff
@@ -170,11 +170,13 @@ extern "C" sig_handler handle_fatal_signal(int sig)
   my_safe_printf_stderr("[ERROR] mysqld got " SIGNAL_FMT " ;\n",sig);
 
   my_safe_printf_stderr("%s",
-    "This could be because you hit a bug. It is also possible that this binary\n"
-    "or one of the libraries it was linked against is corrupt, improperly built,\n"
-    "or misconfigured. This error can also be caused by malfunctioning hardware.\n\n");
-
+                        "Sorry, we probably make a mistake, and this is a bug.\n"
```

Comment:
English is not my native language, but should not that be "made a mistake",  in the past tense?

Reply:
yep it shold be.